### PR TITLE
Fourteen98.information about fbms

### PR DESCRIPTION
**Pull request by Fourteen98**
Added Information regarding the faculty from this page https://www.ktu.edu.gh/academics/faculties/fbms/

### CODE REVIEW

**Comment by Fourteen98** [comment]
Closes #36 #28 #29 #30 #27

File: lambda/lambda_function.py [hunk 1]
```diff
@@ -421,8 +421,106 @@ def handle(self, handler_input):
...
+                .speak(speak_output)
+                # .ask("add a reprompt if you want to keep the session open for the user to respond")
+                .response
+        ) 
+
+class fbmsDepartmentIntentHandler(AbstractRequestHandler):
+    
+    def can_handle(self, handler_input):
+        # type: (HandlerInput) -> bool
+        return ask_utils.is_intent_name("FbmsDepartmentIntent")(handler_input)
```
Comment (smart8099) [inline lambda/lambda_function.py:510]: error, change FbmsDepartmentIntent this to fbmsDepartmentIntentHandler

File: lambda/lambda_function.py [hunk 2]
```diff
@@ -421,8 +421,106 @@ def handle(self, handler_input):
...
+                .speak(speak_output)
+                # .ask("add a reprompt if you want to keep the session open for the user to respond")
+                .response
+        ) 
+
+class fbmsMisionStatementIntentHandler(AbstractRequestHandler):
+    
+    def can_handle(self, handler_input):
+        # type: (HandlerInput) -> bool
+        return ask_utils.is_intent_name("FbmsMisionStatementIntent")(handler_input)
```
Comment (smart8099) [inline lambda/lambda_function.py]: Change FbmsMisionStatementIntent this to the correct name

File: lambda/lambda_function.py [hunk 3]
```diff
@@ -421,8 +421,106 @@ def handle(self, handler_input):
...
+                .speak(speak_output)
+                # .ask("add a reprompt if you want to keep the session open for the user to respond")
+                .response
+        ) 
+
+class fbmsVisionStatementIntentHandler(AbstractRequestHandler):
+    
+    def can_handle(self, handler_input):
+        # type: (HandlerInput) -> bool
+        return ask_utils.is_intent_name("FbmsVisionStatementIntent")(handler_input)
```
Comment (smart8099) [inline lambda/lambda_function.py:470]: change FbmsVisionStatementIntent to the correct handler method name

**Review by smart8099 — CHANGES_REQUESTED** [review]

**Review by smart8099 — APPROVED** [review]

**Comment by Fourteen98** [comment]
Closes #36 #28 #29 #30 #27

**Pull request merged**